frontend/src/components/home/carousel.jsx

```diff
@@ -27,6 +30,63 @@ class MainCarousel extends Component {
             </div>
           </Carousel.Caption>
         </Carousel.Item>
+        <Carousel.Item>
+          <img
+            className="d-block w-100 blurImage"
+            src={ymim5}
+            alt="First Slide"
+          />
+          <Carousel.Caption>
+            <div class="overlay">
+              <h3 className="title-carousel">Young Masterbuilders in Motion</h3>
+              <p className="paragraph-carousel">
+                inspires, connects, and empowers young women orphans, adoptees,
+                and foster youth alumnae to thrive.
+              </p>
+              <Navbar.Brand className="logo-div">
+                <img className="logo" src={logo} alt="YMIM" />
+              </Navbar.Brand>
+            </div>
+          </Carousel.Caption>
+        </Carousel.Item>
```

Comment:
The carousel item seems to be repeated 4 times with different images. Can we abstract this and not repeat code?

Creating a custom CarouselItem component which would be imported into this file would be a great abstraction. However, at a minimum, putting the component within the file is acceptable.

Here's the rough idea:
<img width="585" alt="image" src="https://user-images.githubusercontent.com/906750/69106545-55a9f400-0a34-11ea-96ef-ce4a81433ce1.png">